pages/common/routers/form.js

```diff
@@ -100,6 +100,7 @@ module.exports = ({
     });
     req.form.values = mapValues(req.form.values, trim);
     req.form.values = cleanModel(req.form.values);
+    Object.assign(req.form.values, { ...pickBy(req.body, (field, key) => key.includes('conditional-reveal')) });
```

Comment:
I don't think this is the right way to go about this. It's automatically whitelisting any field with `conditional-reveal`. 

I'd iterate over the schema, and do something like this to get the keys of the associated radio button.

```js
Object.keys(schema).filter(key => schema[key].conditionalReveal).map(key => `conditional-reveal-${key}`)
```

i.e. specifically find the fields which are conditional first, and then map them to their corresponding keys.

Reply:
FINE

Reply:
😢

Reply:
🤗